### DISCUSSION TITLE
Files Downloaded from Edge Browser Not Redirected to 'downloaded_files' Directory

### DISCUSSION THREAD

**Discussion by nuweq** [seleniumbase/SeleniumBase#2578] [Q&A] [answered]
When I try to download files using the Edge browser, they don't end up in the "downloaded_files" folder as expected. Instead, they go to the "C:\Downloads" folder. Even though my script just clicks the download button, this happens. Can you help me fix this? Thanks.

**Comment by mdmintz** [comment, marked as answer]
In order to fix https://github.com/seleniumbase/SeleniumBase/issues/1897 ("Personalize your web experience" notification appears when running on Edge), I set Edge to always use `--guest` mode: https://github.com/seleniumbase/SeleniumBase/issues/1897#issuecomment-1579374303

Edge Guest Mode doesn't allow changing the default download location. However, since extensions don't work in Guest Mode, I added a way to bypass Guest Mode and use normal mode: Add `--disable-csp` / `disable_csp=True`, which will use normal mode, and then files will be correctly saved to the `./downloaded_files/` folder.

**Reply by nuweq** [reply]
Thank you so much for the solution.
